redbot/cogs/trivia/data/lists/minecraft.yml

```
@@ -0,0 +1,114 @@
+AUTHOR: ImajinDevon
+
+"The phantom was added in which update?":
+ - "1.13"
+"Which update introduced a new style combat into the game?":
+ - "1.9"
+'What edition of Minecraft is usually prefixed with the word "bug"?':
+ - "Bedrock"
+"Who created Minecraft?":
+ - "Notch"
+"Who bought Mojang in 2014?":
+ - "Microsoft"
+"Mojang was bought for (...) billion dollars:":
+ - "2.5"
+"If you name a sheep this, it will turn rainbow:":
+ - "jeb_"
```

```suggestion
"If you name a sheep this, its wool will cycle through all the dye colors:":
 - "jeb_"
```

---

I'm going to default back to the wiki's explanation of this easter egg as a guide for how this question should be formed:
> If a sheep is named jeb_, its wool cycles through all dye colors in a similar manner to prismarine. This is purely a visual effect. A sheep named jeb_ can still be dyed without changing the rainbow effect, and any wool obtained from one of these sheep has the most recent color as if the sheep was not named.

```suggestion
"Naming a sheep this will make its wool appear to cycle through all dye colors:":
 - "jeb_"
```